cdhweb/pages/snippets.py

```diff
@@ -248,62 +251,84 @@ def clean(self):
         super().clean()
 
 
-class FooterColumn(ClusterableModel):
+class ImprintLinkItem(MiniMenuItemBase):
     """
-    Represents a column in the footer, which can contain multiple items.
-    Each column has a bilingual heading.
+    Imprint link item for the bottom of the footer.
+    Consists of a title and link to internal or external page.
     """
 
-    class Meta:
-        verbose_name = "Footer column"
-        verbose_name_plural = "Footer columns"
-
-    footer = ParentalKey(
-        "Footer", related_name="footer_columns", on_delete=models.CASCADE
-    )
-    title = models.CharField(  # noqa: DJ001
-        max_length=255, verbose_name="Column heading", blank=True, default=""
+    imprint_link = ParentalKey(
+        "Footer", related_name="imprint_links", on_delete=models.CASCADE
     )
 
-    panels = [
-        FieldPanel("title"),
-        InlinePanel("column_items", label="Column item(s)", max_num=10),
-    ]
+    def __str__(self):
+        return f"Imprint link item: {self.title}"
+
+
+class UsefulLinksItem(MiniMenuItemBase):
+    """
+    Useful link item for the second column of the footer.
+    Consists of a title and link to internal or external page.
+    """
+
+    contact_link = ParentalKey(
+        "Footer", related_name="useful_links", on_delete=models.CASCADE
+    )
 
     def __str__(self):
-        return self.title
+        return f"Useful link item: {self.title}"
 
 
-class ColumnItem(models.Model):
+class ContactLinksItem(models.Model):
     """
-    Represents an item within a footer column, containing a rich text field
-    for contact item body, which supports bold, text, and links.
+    Contact link item for the first column of the footer.
+    Consists of a title and link to internal or external page.
     """
 
-    column = ParentalKey(
-        "FooterColumn", related_name="column_items", on_delete=models.CASCADE
+    contact_link = ParentalKey(
+        "Footer", related_name="contact_links", on_delete=models.CASCADE
     )
+
     body = RichTextField(features=["bold", "link"])
 
     panels = [
         FieldPanel("body"),
     ]
 
-    def __str__(self):
-        if self.title:
-            return self.title
-        return "Column Item"
-
 
-class ImprintLinkItem(MiniMenuItemBase):
+class PhysicalAddress(models.Model):
```

Comment:
What's the motivation for making this a model/inline, rather than just a field?

Reply:
have updated this